protos/hub/hub.proto

```diff
@@ -146,3 +146,13 @@ message Location {
   double lat = 1;
   double lon = 2;
 }
+
+message ChallengeResult {
+  string node_id = 1;
+  string challenge_result = 2;
```

Comment:
should this be an enum? makes range query faster? you can define error messages etc. in a different field

Reply:
yes, and I forgot to close this PR, base on a recent conversation will use Guard proto to store all challenge log.